frameworks/youth-risk-assessment/questions/religious-or-cultural-needs.yml

```diff
@@ -0,0 +1,26 @@
+type: radio
+question: Do they have any religious or cultural needs?
+hint: For example, they may not be eating or drinking for a length of time during specific dates.
+description: Religious or cultural needs
+prefill: true
+options:
+  -
+    label: "Yes"
+    value: "Yes"
+    followup_comment:
+      label: Give details
+      validations:
+        -
+          type: required
+          message: Enter details of any religious or cultural needs
+    flags:
+      -
+        label: Religious or cultural needs
+        type: alert
```

Comment:
should this be `attention` instead?

Reply:
I think it is fine as 'Religious or cultural needs'

Reply:
Agreed as 'attention' (blue) - not red risk flag